packages/manager-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -45,10 +45,10 @@ export const OnboardingLayout: React.FC<OnboardingLayoutProps> = ({
           <div className="flex justify-center py-3 max-h-28">
             <img
```

Comment:
don't you want to conditionally display the image ?

Reply:
until now the onboarding pages that i have seen contains all an image, maybe later but don't need actually to do it for my PR

Reply:
It's not the case, when i dont pass image object as props to the concerned component, it will display the default one by itself   **src={img?.src ?? placeholderSrc}** ( the **placeholderSrc** is the image i need, and the svg is included in the component itself dont need to pass it as props in this case )